community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -644,6 +647,18 @@ def _add_nodesets(yamls: List[Any], target: dict):
         for ns_name in chain(p.partition_nodeset, p.partition_nodeset_dyn, p.partition_nodeset_tpu):
             if ns_name not in ns_names:
                 raise DeffetiveStoredConfigError(f"nodeset {ns_name} not defined in config")
+            
+    for lg_yaml in login_groups:
+        lg_cfg = NSDict(lg_yaml)
+        assert lg_cfg.get("group_name"), "group_name is required"
+        lg_name = lg_cfg.group_name
+        assert lg_name not in cfg.login_groups
+        cfg.login_groups[lg_name] = lg_cfg
+
+    if instance_role() == "login":
+        group = instance_login_group()
+        if group not in cfg.login_groups:
+            raise DeffetiveStoredConfigError(f"login group '{group}' does not exist in config")
```

Comment:
typo for `Defective`?

Reply:
It is, but I wouldn't  address it in this PR.